boolean_algebra/boolean algebra calculator.py

```diff
@@ -0,0 +1,12 @@
+print(
+    "INSTRUCTIONS: USE 'and' / 'or' / 'not' when entering equation. For booleans use True/False or 1/0"
+)
+
+print()
+while True:
+    try:
+        eq = input("Enter equation: ")
+
+        print(eval(eq))
```

Comment:
Please read the file `CONTRIBUTING.md` to understand our definition of an algorithm and our requirements for tests and type hints.

Also, https://nedbatchelder.com/blog/201206/eval_really_is_dangerous.html